app/controllers/bridges/lom/tasks_controller.rb

```diff
@@ -2,20 +2,20 @@
 
 module Bridges
   module Lom
+    # rubocop:disable Metrics/ClassLength
+    # rubocop:disable Metrics/AbcSize
+    # rubocop:disable Metrics/MethodLength
     class TasksController < ActionController::API
```

Comment:
You could try moving the rubocop statements here and/or combine them in a single line:

```
# rubocop:disable Metrics/ClassLength, Metrics/AbcSize, Metrics/MethodLength
```

Reply:
Alright, didn't know that. But I like the simplicity of the one-liner :)